### PR TITLE
Use DigiByte Source docker for basis

### PR DESCRIPTION
**Pull request by ChillingSilence**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+ARG ARCH=x86_64
+
+ARG MAINP2P=12024
+ARG MAINRPC=14022
+ARG TESTP2P=12026
+ARG TESTRPC=14023
+
+# You can confirm your timezone by setting the TZ database name field from:
+# https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
+ARG LOCALTIMEZONE=Pacific/Auckland
```
Comment (gto90) [inline Dockerfile]: @ChillingSilence and @SmartArray , will this time zone have any negative impact on getting time/dates from the node?  Should this be set to UTC?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+# Set to 1 for running it in testnet mode
+ARG TESTNET=0
+
+# Do we want any blockchain pruning to take place? Set to 4096 for a 4GB blockchain prune.
+# Alternatively set size=1 to prune with RPC call 'pruneblockchainheight <height>'
+ARG PRUNESIZE=0
```
Comment (gto90) [inline Dockerfile:24]: For the purpose of keeping this lightweight, should we default the pruning to 1000 blocks or similar @ChillingSilence or @SmartArray ?

**Review by gto90 — COMMENTED** [review]
@SmartArray and @ChillingSilence ,

This is looking good to me.  Just a few comments and questions above and I think we can merge this.

File: Dockerfile (hunk 1 above)
Comment (ChillingSilence) [reply Dockerfile]: It shouldn't really matter but I agree and it can be set to UTC.

File: Dockerfile (hunk 2 above)
Comment (ChillingSilence) [reply Dockerfile:24]: Personally I think it's worth supporting the option of pruning but having the full chain by default while it's still only ~20GB as it is at present. If we got to 1TB+ I'd say it's probably worth reviewing etc down the line but where we can encourage users to run a *full* node I feel it's worthwhile 🙂

Comment (gto90) [reply Dockerfile:24]: I think that sounds fine.  It's easy enough for developers to adjust this setting when building.

File: Dockerfile (hunk 1 above)
Comment (gto90) [reply Dockerfile]: @ChillingSilence can you update this to https://en.wikipedia.org/wiki/Coordinated_Universal_Time ?  I think the code should be:

`ARG LOCALTIMEZONE=UTC`

@SmartArray , thoughts?

Comment (ChillingSilence) [reply Dockerfile]: Updated

**Review by gto90 — APPROVED** [review]

**Pull request merged**